client/src/routes/+layout.server.ts

```diff
@@ -0,0 +1,12 @@
+import type { UserResponseBrief } from "bindings/UserResponseBrief";
+import type { LayoutServerLoad } from "./$types";
+
+export const load = (async ({ fetch }) => {
+  const res = await fetch(`http://localhost:6060/api/v1/users/me`, {
+    credentials: "include",
+  });
+  const user: UserResponseBrief | undefined = res.status !== 200 ? undefined : await res.json();
+  return {
+    user: user
+  };
```

Comment:
1000x performance change

```suggestion
  return { user };
```